hs/src/Reach/Simulator/Core.hs

```diff
@@ -486,8 +494,12 @@ instance Interp DLExpr where
       (g, _) <- getState
       let linstate = e_linstate g
       acc <- vAddress <$> interp dlarg
-      let m = saferMaybe "DLE_MapRef1" $ M.lookup dlmvar linstate
-      return $ saferMaybe "DLE_MapRef2" $ M.lookup acc m
+      case M.lookup dlmvar linstate of
+        Nothing -> return $ V_Maybe V_Nothing
+        Just m -> do
+          case M.lookup acc m of
+            Nothing -> return $ V_Maybe V_Nothing
+            Just m' -> return $ V_Maybe $ V_Just m'
```

Comment:
Ah, yes, these need to return a `V_Data`

Reply:
what should the `SLVar` in `V_Data SLVar DLVal` be here? there are no obvious string values.

Reply:
oh never mind

Reply:
okay!

Reply:
thanks